consensus/module_consensus_pacemaker.go

```diff
@@ -82,7 +82,7 @@ func (m *consensusModule) IsPrepareQCNil() bool {
 func (m *consensusModule) GetPrepareQC() (*anypb.Any, error) {
 	anyProto, err := anypb.New(m.prepareQC)
 	if err != nil {
-		return nil, fmt.Errorf("Failed to convert paceMaker message to proto: %s", err)
```

Comment:
Use `%w` instead of `%s` for errors.

See https://stackoverflow.com/questions/61283248/format-errors-in-go-s-v-or-w#:~:text=Should%20I%20use%20%25s%20or,but%20there%20are%20no%20guarantees.